KK UI bugs

keanderss opened the pull request:
Closes #675

keanderss committed:
Fixed KK migration UI bugs

jaspnas commented on templates/common/members/functionary.html:
```diff
@@ -11,7 +11,7 @@
             <div class="content">
                 <!-- Functionary Table Section -->
                 <h4>Funktionärsposter</h4>
-                <table class="table table-dark table-striped">
```
KK isn't using functionary yet, also will this not break DaTe?

keanderss replied:
Yes, on closer inspection this change would break DaTe I just forgot to check where i was making changes.

jaspnas reviewed (changes requested):
Maybe we should create a default-table CSS class that we use and can define per site

keanderss committed:
Revised previous changes by moving table-light/dark into new table-body class

keanderss committed:
Moved table styles into base.css

keanderss commented on templates/common/members/userinfo.html:
```diff
@@ -20,7 +20,7 @@
                 <h3>{{ user.username }}</h3>
                 <div id="userInfo">
-                    <table class="table table-dark table-striped">
+                    <table id="table" class="table table-striped">
```
Would have preferred to use classes for this but for some reason they were not being applied to DaTe.

keanderss commented:
TODO:
- Fix mobile calendar 
- Fix mobile sponsors

keanderss committed:
Removed old footer and hardcorded sponsors from sidebar. Fixed dynamic logo sizing, mobile calendar and sponsors.

keanderss commented:
For some reason the old footer was not removed from the files which caused the footer issues.

Summary of changes:
- Removed old footer
- Removed hardcoded sponsors (should probably be with the other sponsors)
- Added calendar and sponsors to mobile view
- Fixed dynamic association logo resizing
- Some other minor changes to make KK more consistent with DaTe

keanderss committed:
set cache back to 300

keanderss committed:
Added fallback image for missing sponsors and improved css to prevent logo carousel from breaking

jaspnas commented on templates/kk/core/footer.html:
```diff
@@ -1,54 +0,0 @@
-                <br>
-                20500 Åbo
-                <br>
-                Kanslitid Må - To 11:30-13
```
KK needs their own footer template because they have their "Kanslitid". Maybe it can be a content variable, but maybe too much work to make the common footer dynamic for this if only KK is using it?

jaspnas commented on templates/kk/date/start.html:
```diff
@@ -58,9 +60,7 @@ <h3>{% trans 'Inga nyheter hittades' %}...</h3>
                 </div>
                 <div class="sidebar-container">
                     <div class="events-container">
-                        <a id="andritz-logo" href="https://www.andritz.com/group-en"><img class="sidebar-logo-container" src={% static "/core/images/andritzLogo.png" %} alt="Andritz"></a>
```
These were specifically requested by KK in this form, so let's keep them there.

jaspnas commented on templates/kk/date/start.html:
```diff
@@ -151,8 +214,8 @@ <h3 class="mt-3">Samarbetspartners</h3>
-            // Ensure minutes are two digits
-            minutes = minutes < 10 ? '0' + minutes : minutes;
+            // Ensure hours and minutes are displayed with two digits
+            hours = hours < 10 ? '0' + hours : hours;
```
Do we not need another line here for minutes?

jaspnas commented on templates/date/date/start.html:
```diff
@@ -180,7 +180,7 @@ <h3 class="header-border-bottom">Samarbetspartners</h3>
                             {%for ad in ads%}
                                 <a class="m-2" href="{{ ad.company_url }}">
-                                    <img src="{{ ad.ad_url }}" alt="img">
+                                    <img src="{{ ad.ad_url }}" alt="img" onerror="this.src='/static/core/images/fallback.png'">
```
Nice, maybe add this to Biocum as well after migration.

jaspnas replied:
Probably also want to add this for the mobile container?

jaspnas commented on templates/kk/date/start.html:
```diff
@@ -117,14 +117,77 @@ <h3 class="mt-3">Samarbetspartners</h3>
                         </div>
                     </div>
                 </div>
+                <div class="mobile-events-container">
```
Probably should see if we can fix this without code duplication. Maybe create a new issue to fix it later.

keanderss committed:
Added KK kanslitid to footer

keanderss committed:
Reduce code duplication

keanderss committed:
Reduce code duplication and fix size responsiveness

jaspnas reviewed (approved):
Looks good
